include/godot_cpp/variant/color_names.inc.hpp

```diff
@@ -28,6 +28,11 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#ifndef GODOT_COLOR_NAMES_INC_HPP
+#define GODOT_COLOR_NAMES_INC_HPP
+
+#include <godot_cpp/variant/color.hpp>
```

Comment:
This file is only meant to be included in `color.cpp` and should probably not have the `.hpp` extension, and instead be moved together with `color.cpp`. It's just extra data kept separate for convenience.

It's the same in the main engine:

https://github.com/godotengine/godot/blob/master/core/math/color_names.inc
https://github.com/godotengine/godot/blob/master/core/math/color.cpp#L33

Reply:
Renamed & removed header guards in 0b3b1c4491b332738ac72c7b7d1ccd634b3a852b